pipelines/gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx512m"'
```

Comment:
Add a comment as to why we're increasing this so future folks can debug - perhaps open up a separate issue in GitHub to investigate as well.

Reply:
add comment